### ISSUE TITLE
PrepareInventoryResultEvent deprecated

### ISSUE THREAD

**Issue by SlimeDog** [Shopkeepers/Shopkeepers#820]
## Preliminaries:

* **Shopkeepers version:** 
```
> version Shopkeepers
[22:28:28 INFO]: Shopkeepers version 2.16.4-SNAPSHOT+ad3832a1
[22:28:28 INFO]: Custom villager shopkeepers
[22:28:28 INFO]: Website: https://dev.bukkit.org/projects/shopkeepers/
[22:28:28 INFO]: Authors: nisovin and blablubbabc
```

* **Spigot version:**
```
> version
[22:24:45 INFO]: This server is running Paper version git-Paper-328 (MC: 1.19.3) (Implementing API version 1.19.3-R0.1-SNAPSHOT) (Git: c919e94)
You are running the latest version
```

* I have checked that my issue/question does not get answered by:
  * The [documentation](https://github.com/Shopkeepers/Shopkeepers-Wiki/wiki).
  * The [FAQ](https://github.com/Shopkeepers/Shopkeepers-Wiki/wiki/Frequently-Asked-Questions).
  * The [Known Issues](https://github.com/Shopkeepers/Shopkeepers-Wiki/wiki/Known-Issues).
* I have checked all open and closed [issues](https://github.com/Shopkeepers/Shopkeepers/issues), but none seems to fit my issue/question.

## Reproduction on a fresh and up-to-date Paper server:

I was **`able`** to reproduce my issue on a freshly set up and up-to-date Paper server with the latest version of Shopkeepers, with no other plugins and with no kinds of other server or client mods.

## The issue:

On server start-up
```
[Server thread/WARN]: [Shopkeepers] "Shopkeepers v2.16.4-SNAPSHOT+ad3832a1" has registered a listener for org.bukkit.event.inventory.PrepareInventoryResultEvent on method "void com.nisovin.shopkeepers.shopcreation.CreateListener.onPrepareAnvilEvent(org.bukkit.event.inventory.PrepareAnvilEvent)", but the event is Deprecated. "Server performance will be affected"; please notify the authors [nisovin, blablubbabc].
```

**Comment by blablubbabc** [comment]
Should be fixed in Paper: https://github.com/PaperMC/Paper/pull/8672

**Comment by SlimeDog** [comment]
Confirmed fixed by Paper 1.19.3-330. Thanks for encouraging them to fix it.
